delivery/src/main/java/com/zeecoder/delivery/domain/OrderStatistic.java

```diff
@@ -0,0 +1,18 @@
+package com.zeecoder.delivery.domain;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+import java.util.UUID;
+
+@Document
+@Data
+@Builder
+public class OrderStatistic {
+    private UUID orderId;
```

Comment:
Are you sure all your events will be related to orders? I was thinking about such structure of event
```
{
    eventType: "orderCreated",
    eventDate: "{date in utc}",
    data: {
        // different attributes depending on type like orderId, customerId, itemId etc
    }
}
```